src/cdk/overlay/position/flexible-connected-position-strategy.ts

```diff
@@ -1110,7 +1110,7 @@ export class FlexibleConnectedPositionStrategy implements PositionStrategy {
       return origin.nativeElement.getBoundingClientRect();
     }
 
-    if (origin instanceof HTMLElement) {
+    if (origin instanceof Element) {
```

Comment:
Maybe add a comment here about why it's `Element`, even though the type is for an `HTMLElement`?

Reply:
Done